controllers/database_controller.go

```diff
@@ -55,16 +57,152 @@ const (
 	pxcDeploymentName   = "percona-xtradb-cluster-operator"
 	psmdbDeploymentName = "percona-server-mongodb-operator"
 
-	psmdbCRDName                     = "perconaservermongodbs.psmdb.percona.com"
-	pxcCRDName                       = "perconaxtradbclusters.pxc.percona.com"
-	pxcAPIGroup                      = "pxc.percona.com"
-	psmdbAPIGroup                    = "psmdb.percona.com"
-	haProxyTemplate                  = "percona/percona-xtradb-cluster-operator:%s-haproxy"
-	restartAnnotationKey             = "dbaas.percona.com/restart"
-	ClusterTypeEKS       ClusterType = "eks"
-	ClusterTypeMinikube  ClusterType = "minikube"
+	psmdbCRDName                            = "perconaservermongodbs.psmdb.percona.com"
+	pxcCRDName                              = "perconaxtradbclusters.pxc.percona.com"
```

Comment:
unused?

Reply:
No, this was here before.
https://github.com/percona/dbaas-operator/blob/7c309de83d1e07b9237bb84a50d7ed491d93cf51/controllers/database_controller.go#L748